src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java

```diff
@@ -132,7 +130,12 @@ public FileRawData execute(
         }
     }
 
-    private void enforceFileLimits(Set<DownloadRequest> downloadList, Integer limit, double fileSizeLimit) throws GoGoDuckException {
+    private void enforceFileLimits(SubsetParameters parameters, Set<DownloadRequest> downloadList, Integer limit, double fileSizeLimit) throws GoGoDuckException {
```

Comment:
Maybe we can just pass the boolean parameters.isPointSubset() instead of all the parameters.